.github/workflows/build-m1-binaries.yml

```diff
@@ -0,0 +1,48 @@
+name: Build on M1
+on:
+  pull_request:
+    paths:
+      - .github/workflows/build-m1-binaries.yml
+  push:
+    branches:
+      - nightly
+  workflow_dispatch:
+jobs:
+  build_wheels:
+    name: "Build TorchVision M1 wheels"
+    runs-on: macos-m1
```

Comment:
We have only 1 macos-m1 worker currently. I will change the rest of our workers to adhere to the same macos-m1 label